### PR TITLE
MES 6319 Include Mock Journals When Serving Locally

### PR DESCRIPTION
**Pull request by tomcrawleyy**
## Description
Introduces and fixes the scripts which run when serving locally to generate the required mock data for the journals.
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +61,7 @@
     "cordova-plugin-secure-storage-echo": "^5.1.1",
     "ionic-mocks": "^1.3.0",
     "lodash": "^4.17.20",
-    "moment": "2.24.0",
+    "moment": "^2.29.1",
```
Comment (szabikr) [inline package.json]: What would be the reason for this package upgrade?

**Review by szabikr — COMMENTED** [review]
Can you please add the json files in the `mock` folder and the `src/assets/mock` folder to `.gitignore` please? We don't want to check in those files as they are generated and it'll show up on every PR because the date changed

File: package.json (hunk 1 above)
Comment (tomcrawleyy) [reply package.json]: reverted back

**Comment by tomcrawleyy** [comment]
> Can you please add the json files in the `mock` folder and the `src/assets/mock` folder to `.gitignore` please? We don't want to check in those files as they are generated and it'll show up on every PR because the date changed

Sure thing, all done

**Review by piercemorris — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -14,6 +14,10 @@ UserInterfaceState.xcuserstate
 $RECYCLE.BIN/
 ionic.config.json
 .npmrc
+mock/local-journal.json
+mock/local-journal-non-test-activities.json
+src/assets/mock/local-journal.json
```
Comment (szabikr) [inline .gitignore]: Can you please add the `src/assets/mock` folder all together as it's basically copied from the `mock` folder

**Review by szabikr — APPROVED** [review]

**Pull request merged**